boskos-acquire/0.1/README.md

```diff
@@ -0,0 +1,40 @@
+# Boskos Acquire
+
+The `boskos-acquire` Task can be used to acquire cloud projects from a pool with
+[Boskos](https://github.com/kubernetes-sigs/boskos#boskos), by invoking `boskosctl`.
+
+To release projects obtained with `boskos-acquire` can be released with
+[`boskos-release`](../boskos-release).
+
+It is implemented using [`boskosctl`](https://github.com/kubernetes-sigs/boskos/tree/master/cmd/boskosctl).
+
+_The Task assumes already have Boskos up and running. To set it up yourself, your
+best bet would be to find an example of it already configured, for example
```

Comment:
There is now an example deployment in https://github.com/kubernetes-sigs/boskos/tree/master/deployments/overlays/example, but other docs are still a bit lacking.

Reply:
ill use that example instead!